x-pack/test/functional/apps/dashboard/async_search/async_search.ts

```diff
@@ -23,7 +26,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
 
     it('delayed should load', async () => {
       await PageObjects.common.navigateToApp('dashboard');
-      await PageObjects.dashboard.gotoDashboardEditMode('Delayed 5s');
+      await PageObjects.dashboard.loadSavedDashboard('Delayed 5s');
```

Comment:
@lukasolson, I doubt that `gotoDashboardEditMode` was intentional, so I changed it to `loadSavedDashboard`

`loadSavedDashboard` is faster because we don't have to wait for initial load and then 2nd load after switch to edit mode.